### ISSUE TITLE
NFS5 Track converter?

### ISSUE THREAD

**Issue by TheBorn** [OpenNFS/OpenNFS#33]
I saw some post in issues section regarding this, and although there seem to be some talk about it I have not seen any actual converters or converted tracks? 

So yeah not an issue but rather me asking a question about this.

**Comment by dm1khm** [comment]
Hi @TheBorn . I was able to convert tracks to obj with some minor issues for an attempt to remake NFSPU using UE4. Contact me if you are still looking for a converter.

**Comment by bugsimtasdesimt** [comment]
hello!
the born along with some other people already made a converter for NFSPU

**Comment by TheBorn** [comment]
Yeah I did, but even so there are still issues and things lacking in my stuff too, if you have any idea on how to improve it from its current state, such as getting proper vertex colors and normals I would definitely be interested in it. https://discord.gg/5GECDjUAmS This is the Discord server we made for old NFS reverse engineering developments. You can join here and share your ideas and findings!

**Comment by AmrikSadhra** [comment]
Awesome, great work! :) When I restart the NFS5 portion of this project, you can be sure we'll feed back any findings to you and the server.
